File: extensions/vscode-jetbrains-color-theme/assets/examples/sample.cs
// Single-line comment
/* Multi-line
   comment */
/// XML documentation

namespace MyApp {
    [Serializable]
    public class Person : IDisposable {
        public string Name { get; set; } = "John";
        private int _age;

        public async Task<int> GetDataAsync() {
            using var client = new HttpClient();
            var response = await client.GetAsync("url");
            return (int)response.StatusCode;
        }

        ~Person() {
            Dispose(false);
        }
    }

    delegate void MyDelegate(string msg);

    var query = from num in numbers
                where num > 10
                select num * 2;
}

File: extensions/vscode-jetbrains-color-theme/assets/examples/sample.rs
// Single-line comment
/* Multi-line
   comment */

/// Documentation comment
#[derive(Debug)]
struct Point<T> {
    x: T,
    y: T,
}

impl<T> Point<T> {
    fn new(x: T, y: T) -> Self {
        Point { x, y }
    }
}

async fn fetch_data() -> Result<(), Box<dyn Error>> {
    let response = reqwest::get("url").await?;
    let text = response.text().await?;
    Ok(())
}

fn main() {
    let numbers = vec![1, 2, 3];
    let doubled: Vec<_> = numbers.iter().map(|x| x * 2).collect();
    println!("{:?}", doubled);
}
